tests/Unit/DataStructures/Tensor/Expressions/Test_MixedOperations.cpp

```diff
@@ -64,42 +45,85 @@ G_t compute_expected_result(const R_t& R, const S_t& S, const G_t& G,
   return expected_result;
 }
 
+// Computes the lapse from the inverse spatial metric and spacetime metric
+//
+// \f$N\f$ is calulated using the following equation:
```

Comment:
lapse=alpha, spatial metric = gamma, spacetime metric = g :) We're abandoning the old notation and someone *cough Geoffrey* hasn't updated all the GH notation :)